### ISSUE TITLE
Output cpu, disc and memory information

### ISSUE THREAD

**Issue by thernstig** [sharkdp/hyperfine#711]
Would it be possible for hyperfine to output information about the cpu (type, brand, generation, cores) as well as memory and disc information as part of its results.

It is good information to see if users run the same benchmark on different machines.

**Comment by jjpxiii** [comment]
It would be great ! 
I can try to do this if it helps.

**Comment by sharkdp** [comment]
I would certainly like to see this being part of the JSON export. Maybe we can also add it to the terminal output, but only behind a feature flag. What do you think?

**Comment by thernstig** [comment]
Yes, part of the CLI in one way or another was my initial idea, so thumbs up for that.

**Comment by AbdulfatahMohammedSheikh** [comment]
fix this in #783
